### ISSUE TITLE
Error in examples/solve_with_bp.jl

### ISSUE THREAD

**Issue by WPettersson** [jeremyomer/KidneyExchange.jl#23]
With latest master I get the following. I suspect the re-working of file IO has broken this example, but I'm also not sure what the intent of the latter part of this file.

If I comment out https://github.com/jeremyomer/KidneyExchange.jl/blob/e8fd74820acdbccf3f07348dce7bc183bf02f9e3/examples/solve_with_bp.jl#L43 and any of the lines at or following https://github.com/jeremyomer/KidneyExchange.jl/blob/e8fd74820acdbccf3f07348dce7bc183bf02f9e3/examples/solve_with_bp.jl#L50 the file seems to run again. 

Error message:
```
% ~/opt/julia-1.10.1/bin/julia examples/solve_with_bp.jl
ERROR: LoadError: MethodError: no method matching read_wmd_file(::String)

Closest candidates are:
  read_wmd_file(::Any, ::Any)
   @ KidneyExchange ~/.julia/packages/KidneyExchange/07zMq/src/instance/matching_instance.jl:205

Stacktrace:
 [1] top-level scope
   @ ~/src/git/KidneyExchange.jl/examples/solve_with_bp.jl:43
in expression starting at /home/wpette/src/git/KidneyExchange.jl/examples/solve_with_bp.jl:43
```

**Comment by pnavaro** [comment]
Hi William, I had some explanation by the original authors of the package. They told me that the preflib file format changed during the developpement of the package and they did not want to upgrade it because their results in the paper were made with the old format. I tried to manage this new preflib format and keep the old behaviour for people who want to reproduce the paper simulations. That's why I created this example.

If you run julia with MD files in the working directory, it works. I fixed the path to these files in #24
